examples/websites/README.md

```diff
@@ -0,0 +1,53 @@
+#ipfs for websites
+### A short guide to hosting your site on ipfs
+
+Adding your static website to ipfs is quite simple! Simply turn on your daemon:
+```bash
+$ ipfs daemon
+```
+
+And add the directory containing your website:
+```bash
+$ ls mysite
+index.html
+$ ipfs add -r mysite
+added QmbRyftGAtNjXs7hWZUTLyp27qrpgRUXKEwXCVPBueZGek mysite/index.html
+added QmYAMp9ptyn3Bv8ijoYoYAAQHFCfpZ1NKRNedkEWrR9DPH mysite/
```

Comment:
- the ipfs.io has a lot going on. ideally it would be a small site.
- spacecat \o/ -- cats + space = perfect for ipfs. the internet in space = look at cats from mars.